builds/e2e/templates/deploy-iotuap.yaml

```diff
@@ -128,20 +102,12 @@ steps:
           -E2ETestFolder "C:\Data\e2etests" `
           -ReleaseLabel "${{ parameters['release.label'] }}" `
           -ArtifactImageBuildNumber "$($ArtifactInfo.BuildNumber)" `
-          -TestName "${{ parameters['testName'] }}" `
+          -TestName "LongHaul" `
           -ContainerRegistry "${{ parameters['container.registry'] }}" `
           -ContainerRegistryUsername "${{ parameters['container.registry.username'] }}" `
           -ContainerRegistryPassword "${{ parameters['container.registry.password'] }}" `
           -IoTHubConnectionString "${{ parameters['iotHub.connectionString'] }}" `
           -EventHubConnectionString "${{ parameters['eventHub.connectionString'] }}" `
-          -SnitchBuildNumber "${{ parameters['snitch.build.number'] }}" `
-          -SnitchStorageAccount "${{ parameters['snitch.storage.account'] }}" `
-          -SnitchStorageMasterKey "${{ parameters['snitch.storage.masterKey'] }}" `
-          -SnitchAlertUrl "${{ parameters['snitch.alert.url'] }}" `
-          -TransportType1 "${{ parameters['transportType1'] }}" `
-          -TransportType2 "${{ parameters['transportType2'] }}" `
-          -TransportType3 "${{ parameters['transportType3'] }}" `
-          -TransportType4 "${{ parameters['transportType4'] }}" `
```

Comment:
Seems weird to me that these are being passed in. Unused?

Reply:
Yeah, seemed strange to me also. Looking closely at the logic, they seemed to be unused.